src/styles/index.ts

```diff
@@ -161,6 +164,7 @@ export const GlobalStyles = createGlobalStyle`
       --button-secondary-section: #16181E;
       --button-boder: rgba(255, 255, 255, 4%);
       --shadow-default: rgba(230, 232, 242, 0.05);
+      --shadow-tooltip: 0px 4px 6px -2px rgba(16, 24, 40, 0.03), 0px 12px 16px -4px rgba(16, 24, 40, 0.08), 0 0 8px rgba(230, 232, 242, 0.05);
```

Comment:
Let's name it without context, please. This shadow is used in many places.

```suggestion
      --shadow-box: 0px 4px 6px -2px rgba(16, 24, 40, 0.03), 0px 12px 16px -4px rgba(16, 24, 40, 0.08), 0 0 8px rgba(230, 232, 242, 0.05);
```